src/main/java/net/sf/jabref/logic/cleanup/CleanupPreset.java

```diff
@@ -0,0 +1,210 @@
+/*  Copyright (C) 2003-2015 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+package net.sf.jabref.logic.cleanup;
+
+public class CleanupPreset {
```

Comment:
What do you think about a hybrid wrapper similar to #658 ? That is, still store the boolean values in a class (with "state"), but move the methods to load/save from the preferences also to this new CleanupPreset class. This has the advantage that the Cleanup process has no dependency on the preferences and it brings some performance improvements with it (no 10*number of entries calls to getBoolean).